lib/driver.js

```diff
@@ -484,6 +482,37 @@ export class XCUITestDriver extends BaseDriver {
     }
   }
 
+  async handleMjpegOptions() {
+    const mjpegServerPort = this.opts.mjpegServerPort || DEFAULT_MJPEG_SERVER_PORT;
+    await this.allocateMjpegServerPort(this.opts.udid,
+      mjpegServerPort, this.isRealDevice() ? true : false);
+    // turn on mjpeg stream reading if requested
+    if (this.opts.mjpegScreenshotUrl) {
+      this.log.info(`Starting MJPEG stream reading URL: '${this.opts.mjpegScreenshotUrl}'`);
+      this.mjpegStream = new mjpeg.MJpegStream(this.opts.mjpegScreenshotUrl);
+      await this.mjpegStream.start();
+    }
+  }
+
+  async allocateMjpegServerPort(udid, mjpegServerPort, usePortForwarding) {
```

Comment:
all arguments here are available as instance properties

Reply:
also consider providing doctring with explicit types for function arguments as well as for its return type. This helps the ts compiler to detect silly typing mistakes